docs/includes/topic_f5lbaas-l7_content_switching.rst

```diff
@@ -218,9 +220,9 @@ Configuration
 Further Reading
 ---------------
 
```

Comment:
Looks like a footnote is special markup in the text. If we call it out without a preceding '..' before it, in the way you have above, it gives us an error of:

`
Too many autonumbered footnote references: only 0 corresponding footnotes available.
`

I modified it to have seealso references, then to have the footnotes at the very bottom.